dev/NavigationView/NavigationView.xaml

```diff
@@ -403,11 +447,13 @@
                                         BorderThickness="0,0,1,0">
                                         <Grid.RowDefinitions>
                                             <RowDefinition Height="Auto"/>
-                                            <RowDefinition Height="0"/> <!-- above button margin + back button space -->
+                                            <RowDefinition Height="0"/>
+                                            <!-- above button margin + back button space -->
```

Comment:
Not sure what the comment is referring too... Is it the the PaneContentGridToggleButtonRow? But is that really a margin?

Reply:
No idea, but I reverted the comment to be inline.